tests/unit/backup/backends/v3/test_backend.py

```diff
@@ -33,33 +33,23 @@ def test_writer_reader_roundtrip(tmp_path: Path) -> None:
     finished_at = datetime.datetime.now(datetime.timezone.utc)
 
     records = (
-        ConsumerRecord(
+        StubMessage(
             key=b"foo",
             value=b"bar",
             topic=topic_name,
             partition=partition_index,
             offset=10,
-            timestamp=round(time.time()),
-            timestamp_type=None,
-            headers=(),
-            checksum=None,
-            serialized_key_size=None,
-            serialized_value_size=None,
-            serialized_header_size=None,
+            timestamp=(1, round(time.time())),  # TODO
```

Comment:
nit (non-blocking): Are these todos for the timestamp type? Maybe better to remove them since we have a ticket now? Otherwise should probably have more context. (2 more in same file)

Reply:
Good catch, I missed these after introducing `Timestamp`